rio-backend/src/ansi/mod.rs

```diff
@@ -27,6 +34,16 @@ impl CursorShape {
     }
 }
 
+impl From<CursorShape> for char {
+    fn from(value: CursorShape) -> Self {
+        match value {
+            CursorShape::Underline => '_',
+            CursorShape::Beam => '|',
+            _ => '▒',
```

Comment:
I think the character is `'▇'`

Reply:
actually we can get rid of this `From<CursorShape> for char`

Reply:
and doesn't matter though which character, since it always fallbacks to block

Reply:
58b0c1b3094f1085e12a7827ab7e08495fd36aa3